config/jobs/kubernetes-sigs/testing_frameworks/testing-frameworks-config.yaml

```diff
@@ -5,15 +5,32 @@ presubmits:
     - master
     always_run: true
     skip_report: false
+    decorate: true
```

Comment:
Decorate will inject init/sidecar containers into your spec which cause this job to:
* checkout the repo, merging in any PRs (so you don't need to use `bootstrap.py --repo`
* upload your results gcs/testgrid (so you don't need to `bootstrap.py --upload`)

If you use decoration you can assume that your container will start in the http://github.com/kubernetes-sigs/testing_frameworks directory